app/src/main/java/com/example/simonhsieh/ddcharactersheet/model/Attributes.java

```diff
@@ -0,0 +1,52 @@
+package com.example.simonhsieh.ddcharactersheet.model;
+
+/**
+ * Created by simonhsieh on 3/12/18.
+ */
+
+public class Attributes {
+    private int _strength;
```

Comment:
Is underscore variables a JavaScript thing? Just curious.

Reply:
uh yup but I'm gonna delete this shit anyways